pyuntl/untldoc.py

```diff
@@ -675,3 +676,39 @@ def find_untl_errors(untl_dict, **kwargs):
         'error_dict': error_dict,
     }
     return found_data
+
+
+def untl_dict_to_tuple(untl_dict):
+    """Convert untl_dict values to list of lists of tuples."""
+    untl_tuple = {}
+    for elem, value in untl_dict.items():
+        for i, v in enumerate(value):
+            # We are trying to get a consistent ordering of values
+            # so reordering doesn't count as a change.
+            if isinstance(value[i]['content'], dict):
+                value[i]['content'] = [v for v in value[i]['content'].items()]
+                value[i]['content'].sort()
+            value[i] = [v for v in value[i].items()]
```

Comment:
I think it's fine to leave the code as you have it here, though now that you made the most recent change, I notice that this could be:
```
            if isinstance(value[i]['content'], dict):
                value[i]['content'] = sorted(list(value[i]['content'].items()))
            value[i] = list(value[i].items())
```
Although, I am also now wondering if we should be sorting on the last line too to guarantee the same hash when comparing to another dictionary of hash values that comes from untl elements... Do you think that makes sense?

Reply:
yes, by doing that `content` tuple will be in the list, followed by `qualifier` tuple, so that if the `content` and `qualifier` ordering changes in input untl_dict, the hash still remains the same.